api/src/domain/license-status/searchLicenseStatusFactory.ts

```diff
@@ -1,18 +1,19 @@
 import { SearchLicenseStatus } from "@domain/types";
 
-export const searchLicenseStatusFactory = (
-  legacySearchLicenseStatus: SearchLicenseStatus,
-  newSearchLicenseStatus: SearchLicenseStatus
-) => {
+export const searchLicenseStatusFactory = (legacy: SearchLicenseStatus, updated: SearchLicenseStatus) => {
```

Comment:
[sand] I don't know if I like `legacy` and `new`/`updated` as the names for these. Legacy as a term just bothers me because it implies that it's intended to be phased out and replaced. We more than likely will continue to rely on MLO more in the coming sprints.

In `app.ts` where we invoke this function, the positional params that get passed in are: `webserviceLicenseStatusProcessorClient` and `dynamicsLicenseStatusClient`. Thoughts on using "webservice" and "dynamics" as part of the variable names?

Reply:
ill use `webservice` and `dynamics` as variable names